imagersite/imager_images/templatetags/images_filters.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from django import template
+from imager_images.models import Photo, Album
+from django.db.models import Q
+
+
+register = template.Library()
+
+
+@register.filter
+def authorized_albums(albums, user):
+    return albums.filter(
+        Q(user=user) | Q(published='shared') | Q(published='public')
+    ).all()
+
+
+@register.filter
+def authorized_photos(photos, user):
+    return photos.filter(
+        Q(user=user) | Q(published='shared') | Q(published='public')
+    ).all()
+
+
+@register.filter
+def album_is_authorized(album, user):
+    # return album in Album.objects.filter(
+    #     Q(user=user) | Q(published='shared') | Q(published='public')
+    # ).all()
+    return album.published in ['shared', 'public'] or album.user.id == user.id
```

Comment:
Nice use of custom filters. Try to keep the code clean by removing old code that is commented out when pulling to master. If you really need to get it back later, you can just refer to past commits where it was present.